siege_utilities/identifiers/__init__.py

```
@@ -0,0 +1,57 @@
+    attestation_uuid,
+    uuid5_from_seed,
+)
+
+__all__ = [
+    "derive_root",
+    "derive_sub_namespace",
+    "uuid5_from_seed",
+    "attestation_uuid",
+    "normalize_name_v1",
+    "NORMALIZER_VERSIONS",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)